pkg/api/controller.go

```diff
@@ -854,6 +858,129 @@ func (c *Controller) StsLogin(w http.ResponseWriter, r *http.Request, body apige
 	writeResponse(w, r, http.StatusOK, responseToken)
 }
 
+func (c *Controller) GetTokenRedirect(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	// Login method needs no auth!
+	redirect, err := c.loginTokenProvider.GetRedirect(ctx)
+	if c.handleAPIError(ctx, w, r, err) {
+		return
+	}
+
+	w.Header().Set("Location", redirect.RedirectURL)
+	w.Header().Set("X-LakeFS-Mailbox", redirect.Mailbox)
```

Comment:
The header value already referenced more than once in this PR, can you please add it to a const (for example [here](https://github.com/treeverse/lakeFS/blob/master/pkg/api/apiutil/constant.go))

Reply:
Done.  But I put it in httputil.  AFAIU apiutil is for values we pass to the lakeFS API, httputil is for values we pass to http.